### PR TITLE
hero image

### LINKED ISSUE #58: Homepage - Add Mobile Hero Image

### PR DESCRIPTION
**Pull request by kcbreuil**
closes #58

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "[javascript]": {
-        "editor.defaultFormatter": "esbenp.pretter-vscode",
+        "editor.defaultFormatter": "esbenp.prettier-vscode",
```
Comment (asasmith) [inline .vscode/settings.json:3]: huh, I guess this is why this wasn't working before?

File: src/pages/index.js [hunk 2]
```diff
@@ -23,58 +23,87 @@ const content = {
...
         },
     },
-}
+};
 
 export default function IndexPage({ data }) {
-    const { heroImage, accImage } = data;
+    const { heroImage, mobileHero, accImage } = data;
 
+    console.log(data);
```
Comment (asasmith) [inline src/pages/index.js:46]: remove this `console.log()` please

File: src/pages/index.js [hunk 3]
```diff
@@ -23,58 +23,87 @@ const content = {
...
+        accImage: file(relativePath: { regex: "/homepageAcc/" }) {
             childImageSharp {
                 fluid(maxWidth: 2000) {
                     ...GatsbyImageSharpFluid
                 }
             }
         }
+        mobileHero: file(relativePath: { regex: "/mobileHero/" }) {
+            childImageSharp {
+                fluid(maxWidth: 600) {
```
Comment (asasmith) [inline src/pages/index.js:94]: Let's increase the `maxWidth` to `768` to match the media query.

**Review by asasmith — COMMENTED** [review]
also add that this closes #61

File: .vscode/settings.json (hunk 1 above)
Comment (kcbreuil) [reply .vscode/settings.json:3]: lol @asasmith so weird

**Pull request merged**